src/coordinate.cpp

```diff
@@ -73,8 +73,8 @@ void test_coordinate() {
       {
         c.reset_coords_to_zero();
         c.move(forward_direction, speed);
-        const double error_x = abs(get_x(c) - speed);
-        const double error_y = abs(get_y(c) - 0);
+        const double error_x = fabs(get_x(c) - speed);
+        const double error_y = fabs(get_y(c) - 0);
```

Comment:
Ah, good find! Next time, use `std::abs` instead. The reason that `abs` works is C backwards compatibility. I've created the junior Issue #734 for just that.